README.md

```
@@ -0,0 +1,55 @@
+# Legion Command Center
+
+**A gamified fitness coaching business platform for the tech-savvy generation**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)